src/pages/home/report/ThreadDivider.tsx

```diff
@@ -47,6 +50,16 @@ function ThreadDivider({ancestor, isLinkDisabled = false}: ThreadDividerProps) {
                 <PressableWithoutFeedback
                     onPress={() => {
```

Comment:
This function and the function in https://github.com/Expensify/App/blob/aa396dbf7c3fd94f1dc26c4e26a1415f61c3bdf5/src/pages/home/report/ReportActionItemParentAction.tsx#L138 are identical. Can we move this logic into a single function that is used by both files? 

Checklist item
>  I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)